mentors/java/a.json

```diff
@@ -54,5 +54,13 @@
     "link_url": null,
     "avatar_url": null,
     "bio": "I have been working in Java for the last 13 years at a software major. Beside my everyday work, I am also responsible for teaching Java (Core and J2EE) to new joinees."
+  },
+  {
+    "github_username": "calmcoconut",
```

Comment:
Your GitHub account name starts with `c`.  You should be adding your bio to c.json not a.json.